### PR TITLE
Java bindings: make final jar publishable

### PR DESCRIPTION
**Pull request by bzz**
This will allow to run `./sbt publish-local` to get latest version of enry-java with native dependencies and JNI installed on local machine.

### CODE REVIEW

**Comment by bzz** [comment]
Can not add reviewers in Github UI
\cc @abeaumont @erizocosmico for a review

File: README.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ $ enry --json
 
 Note that even if enry's CLI is compatible with linguist's, its main point is that, contrary to linguist, **_enry doesn't need a git repository to work!_**
 
+Java bindings
+------------
+
+Generated Java binidings using a C shared library + JNI are localted under [`./java`](https://github.com/src-d/enry/tree/master/java)
```
Comment (abeaumont) [inline README.md]: * s/localted/located/
* Is absolute url used on purpose?

Comment (bzz) [inline README.md]: Absolute url was used due to lack of knowledge of Github features to have relative urls :)

**Comment by bzz** [comment]
History re-written, review addressed in b73364d

**Comment by abeaumont** [comment]
Thanks @bzz !

**Comment by bzz** [comment]
Thank you for the prompt feedback!

**Pull request merged**